### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by hassanekn**
ID: 30886696

### CODE REVIEW

File: res/values-ar/strings.xml [hunk 1]
```diff
@@ -1272,8 +1272,7 @@
     <string name="You_cannot_play_for_x_minutes_">لاتستطيع اللعب لمُدَّة %d دقيقة.</string>
     <string name="x4_Clan_XP_Perm">4x نقاط خبرة عشيرة دائم!</string>
     <string name="x4_Clan_XP_7_days">4x نقاط خبرة العشيرة لمُدَّة 7 ايَّام.</string>
-    <string name="x4_Clan_XP_1_Day">4x نقاط خبرة العشيرة لمُدَّة يوم</string>
-    <string name="CHARGE_UP">الشُحنْةُ</string>
+    <string name="x4_Clan_XP_1_Day">4x نقاط خبرة العشيرة لمُدَّة يوم</string>   <string name="CHARGE_UP">الشُحنْةُ</string>
```
Comment (Jeii20) [inline res/values-ar/strings.xml:1275]: What happened here?

**Review by Jeii20 — CHANGES_REQUESTED** [review]

**Comment by hassanekn** [comment]
Nothing happened, this happened by mistake

في الأربعاء، ٤ سبتمبر ٢٠٢٤, ٦:٢٥ م Jeii20 ***@***.***> كتب:

**Comment by Jeii20** [comment]
Then could you please fix it? All you need to do is to make a line break for the next `<string>`, just like it was before + perfectly aligned with the others. @hassanekn

**Comment by Jeii20** [comment]
It still keeps:

**Comment by hassanekn** [comment]
It should be fixed now.

في الأربعاء، ٤ سبتمبر ٢٠٢٤, ٧:٠٨ م Jeii20 ***@***.***> كتب:

**Review by Jeii20 — APPROVED** [review]
Perfect, thanks for your time. ^^
Already approved, can merge into main as soon as possible.

**Comment by hassanekn** [comment]
You're welcome!

في الأربعاء، ٤ سبتمبر ٢٠٢٤, ٩:٥٥ م Jeii20 ***@***.***> كتب:

**Comment by hassanekn** [comment]
Sorry but how can I merge it

في الأربعاء، ٤ سبتمبر ٢٠٢٤, ١٠:٢٦ م حسن آل برناوي ***@***.***>
كتب:

**Pull request closed without merging**